docs/SETUP.md

```diff
@@ -0,0 +1,74 @@
+# Setup
+
+## Preparation
+
+In order to allow ecChronos to run there are a few tables that needs to be present.
```

Comment:
This sounds like creation isn't automatic. Try: "In order to allow ecChronos to run there are a few tables that are created."

Reply:
After some offline discussions I figured out that the creation is not automatic. I believe we should make this automatic. I understand that we need to face the UUID problem of creating tables concurrently, but we could simply include a script or something that can be run afterwards as a compromise.

Either way, this should not be solved in this patchset.